src/Functions.php

```diff
@@ -12,95 +12,111 @@
+    /**
+     * Runs the given closure before each test in the current file.
+     *
+     * @return BeforeEachCall|TestCase|mixed
+     */
+    function beforeEach(Closure $closure = null): BeforeEachCall
+    {
+        $filename = Backtrace::file();
+
+        return new BeforeEachCall(TestSuite::getInstance(), $filename, $closure);
+    }
 }
 
-/**
- * Registers the given dataset.
- *
- * @param Closure|iterable<int|string, mixed> $dataset
- */
-function dataset(string $name, $dataset): void
-{
-    Datasets::set($name, $dataset);
+if (!function_exists('dataset')) {
+    /**
+     * Registers the given dataset.
+     *
+     * @param Closure|iterable<int|string, mixed> $dataset
+     */
+    function dataset(string $name, $dataset): void
+    {
+        Datasets::set($name, $dataset);
+    }
 }
 
-/**
- * The uses function binds the given
- * arguments to test closures.
- */
-function uses(string ...$classAndTraits): UsesCall
-{
-    $filename = Backtrace::file();
+if (!function_exists('uses')) {
+    /**
+     * The uses function binds the given
+     * arguments to test closures.
+     */
+    function uses(string ...$classAndTraits): UsesCall
+    {
+        $filename = Backtrace::file();
 
-    return new UsesCall($filename, $classAndTraits);
+        return new UsesCall($filename, $classAndTraits);
+    }
 }
 
-/**
- * Adds the given closure as a test. The first argument
- * is the test description; the second argument is
- * a closure that contains the test expectations.
- *
- * @return TestCall|TestCase|mixed
- */
-function test(string $description = null, Closure $closure = null)
-{
-    if ($description === null && TestSuite::getInstance()->test !== null) {
-        return new HigherOrderTapProxy(TestSuite::getInstance()->test);
+if (!function_exists('test')) {
+    /**
+     * Adds the given closure as a test. The first argument
+     * is the test description; the second argument is
+     * a closure that contains the test expectations.
+     *
+     * @return TestCall|TestCase|mixed
+     */
+    function test(string $description = null, Closure $closure = null)
+    {
+        if ($description === null && TestSuite::getInstance()->test !== null) {
+            return new HigherOrderTapProxy(TestSuite::getInstance()->test);
+        }
+
+        $filename = Backtrace::testFile();
+
+        return new TestCall(TestSuite::getInstance(), $filename, $description, $closure);
     }
-
-    $filename = Backtrace::testFile();
-
-    return new TestCall(TestSuite::getInstance(), $filename, $description, $closure);
 }
 
-/**
- * Adds the given closure as a test. The first argument
- * is the test description; the second argument is
- * a closure that contains the test expectations.
- *
- * @return TestCall|TestCase|mixed
- */
-function it(string $description, Closure $closure = null): TestCall
-{
-    $description = sprintf('it %s', $description);
-
-    return test($description, $closure);
+if (!function_exists('')) {
```

Comment:
Isn't that a mistake that the '' is empty? Or am I reading something wrong?

Reply:
haha, thanks a lot! Great catch. 

I made a quick autohotkey for this, guess I missclicked here 😆